exporters/prometheus/exporter.go

```diff
@@ -87,7 +90,7 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 
 	// TODO(#3166): convert otel resource to target_info
 	// see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#resource-attributes-1
-	for _, metricData := range getMetricData(metrics) {
+	for _, metricData := range c.getMetricData(metrics, c.withoutUnits) {
```

Comment:
This is passing a field to the method of the same value type. It looks like this was a hold over from a previous approach. Should it be removed?

Reply:
removed.